scripts/obtain/dotnet-install.sh

```diff
@@ -84,6 +84,10 @@ get_os_download_name_from_platform() {
             echo "opensuse.42.1"
             return 0
             ;;
+        "rhel.6"*)
```

Comment:
@johnbeisner please take a look at these changes.

Reply:
@janvorli
This identification is used for legacy assets only. Since we did not produce specific legacy "rhel.6" assets, I cannot see how this would work. Have you tested this?

Reply:
We will be producing rhel.6 specific assets. rhel.6 is not compatible with the portable Linux stuff. 
And it is not the case anymore that it is used only for legacy assets. I have added usage of this function to get_current_os_name so that I can get rhel.6 out of it. The file we download from Azure is dotnet-sdk-2.1.0-preview1-007172-**rhel.6**-x64.tar.gz, I have already placed it there.

Reply:
We have stated that Linux portable does not work on RHEL.6 - do we know if the assets for RHEL.7 [tar & rpm] fail on RHEL.6?
Examples:
https://dotnetcli.blob.core.windows.net/dotnet/Sdk/2.1.0-preview1-007172/dotnet-sdk-2.1.0-preview1-007172-rhel-x64.rpm
https://dotnetcli.blob.core.windows.net/dotnet/Sdk/1.1.4/dotnet-dev-rhel-x64.1.1.4.tar.gz

Reply:
Yes, they fail because they refer to symbols in glibc that don't exist in the glibc on RHEL 6. There are also some missing features in the kernel on RHEL 6. RPMs would not be able to install at all since there are missing packages that don't exist for RHEL 6.